lib/rubocop/cop/variable_force.rb

```diff
@@ -55,6 +55,27 @@ class VariableForce < Force # rubocop:disable Metrics/ClassLength
 
       SEND_TYPE = :send
 
+      handlers = {}
+
+      handlers[VARIABLE_ASSIGNMENT_TYPE] = :process_variable_assignment
```

Comment:
I wonder if this huge block of init code shouldn't be moved to some class method.